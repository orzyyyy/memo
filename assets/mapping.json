[{"id":"57ca721de1c14fb189d8bc5f6e14448c","title":"改 cookie 步骤","createTime":1560220086082,"modifyTime":1570929537383,"type":"编程","subType":"ex-hentai","category":"markdown"},{"id":"d84c686eca77b9db11063f995d4f4694","title":"deploy with gh-pages and circleci","createTime":1560559934239,"modifyTime":1568419907837,"type":"编程","subType":"其他","category":"markdown"},{"id":"5652a52ab188e25bddf687b32d4ce326","title":"configurate after install","createTime":1561623131393,"modifyTime":1576627065241,"type":"编程","subType":"linux","category":"markdown"},{"id":"b921eb06421a6a25e75c8009721d7324","title":"github ssh proxy","createTime":1561705415318,"modifyTime":1576220297945,"type":"编程","subType":"linux","category":"markdown"},{"createTime":1562317110304,"modifyTime":1568418967118,"id":"91ab070449395f41409a2dd206c25c6b","title":"settings sync 同步","type":"编程","subType":"其他","category":"markdown"},{"createTime":1568944473443,"modifyTime":1575168868918,"id":"fc5fbc1f7dbcbcc13d7ff435776034b1","title":"种类分布","type":"经营","subType":"材料","category":"mapping"},{"createTime":1571276218886,"modifyTime":1571617555436,"id":"9681c099c110874ff452724195c13dea","title":"他和她们、我和他的故事","type":"其他","subType":"练笔","category":"markdown"},{"createTime":1572524120815,"modifyTime":1577343434374,"id":"76f663e411865e7811cf56576abbd3e6","title":"实施记录","type":"经营","subType":"记录","category":"markdown"},{"createTime":1573431918741,"modifyTime":1575848093498,"id":"867507c5d6c8e4a29ed2b5151a9f9d6a","title":"对单一表单中大量状态进行分类重构的记录","type":"编程","subType":"其他","category":"markdown"},{"createTime":1573696315550,"modifyTime":1573699216154,"id":"01859af5536c921096cae77a99ca7bf1","title":"对部分人文价值观的记录","type":"其他","subType":"自省","category":"markdown"},{"createTime":1574469371570,"modifyTime":1577965037044,"id":"35a5e7e5488c9804d0197640baf77dab","title":"pac list","type":"其他","subType":"记录","category":"markdown"},{"createTime":1578195046418,"modifyTime":1578195136167,"id":"588f9fc26caf723542bce305653da31b","title":"2012-07-23","type":"其他","subType":"记录","category":"markdown"},{"createTime":1578195205516,"modifyTime":1578195205516,"id":"daa1c5c8cfac2e4192dfd10c80b0d030","title":"2012-07-24","type":"其他","subType":"记录","category":"markdown"},{"createTime":1578195809385,"modifyTime":1578195809385,"id":"6846cd3418a377ebbc16be347c09eea2","title":"2015-01-08","type":"其他","subType":"记录","category":"markdown"},{"createTime":1578195983510,"modifyTime":1578195983510,"id":"517d2b798dc807c3273e4a09a1b011ab","title":"2015-07-23","type":"其他","subType":"记录","category":"markdown"},{"createTime":1578449803319,"modifyTime":1578449803319,"id":"67b3a6035957c9c1a0980e4710fd1771","title":"7 x 2 碎图","type":"","subType":"","category":"utils"}]
