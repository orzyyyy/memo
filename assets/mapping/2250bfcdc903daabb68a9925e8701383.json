{"CanvasPosition":{"x":-2989,"y":-3560,"z":0,"gap":1},"BlockGroup":{"block-22683":{"x":3843,"y":3572},"block-72108":{"x":3408,"y":3816},"block-391416":{"x":3172,"y":3990},"block-404318":{"x":3614,"y":3989},"block-620761":{"x":3441,"y":4169},"block-624523":{"x":3594,"y":4169},"block-677164":{"x":3744,"y":4171},"block-756260":{"x":3427,"y":4357},"block-759655":{"x":3544,"y":4358},"block-768369":{"x":3663,"y":4357},"block-779054":{"x":3301,"y":4358},"block-785673":{"x":3797,"y":4357},"block-830615":{"x":3174,"y":4357},"block-906652":{"x":4268,"y":3815},"block-4900":{"x":3984,"y":3985},"block-151421":{"x":4169,"y":3986},"block-169247":{"x":4350,"y":3986},"block-198629":{"x":4528,"y":3986},"block-212716":{"x":3982,"y":4169},"block-243049":{"x":4167,"y":4172},"block-294569":{"x":4348,"y":4176},"block-314275":{"x":4526,"y":4175}},"TagGroup":{"tag-915462":{"x":3864,"y":3600,"editable":false,"input":"主旨观点"},"tag-380909":{"x":3435,"y":3844,"editable":false,"input":"关键字"},"tag-394965":{"x":3201,"y":4018,"editable":false,"input":"高频词"},"tag-412650":{"x":3647,"y":4017,"editable":false,"input":"连接词"},"tag-633879":{"x":3476,"y":4198,"editable":false,"input":"强调"},"tag-641204":{"x":3630,"y":4200,"editable":false,"input":"总结"},"tag-699303":{"x":3770,"y":4199,"editable":false,"input":"对应关系"},"tag-793279":{"x":3336,"y":4387,"editable":false,"input":"转折"},"tag-799624":{"x":3462,"y":4387,"editable":false,"input":"递进"},"tag-806051":{"x":3577,"y":4387,"editable":false,"input":"因果"},"tag-813535":{"x":3698,"y":4388,"editable":false,"input":"条件"},"tag-818882":{"x":3835,"y":4386,"editable":false,"input":"假设"},"tag-836778":{"x":3205,"y":4388,"editable":false,"input":"并列"},"tag-972596":{"x":4291,"y":3844,"editable":false,"input":"行文结构"},"tag-33243":{"x":3778,"y":3660,"editable":false,"input":"这段文字意在说明、这段文字说的是"},"tag-135492":{"x":4018,"y":4014,"editable":false,"input":"总分"},"tag-159120":{"x":4206,"y":4014,"editable":false,"input":"分总"},"tag-183740":{"x":4388,"y":4013,"editable":false,"input":"并列"},"tag-204830":{"x":4566,"y":4015,"editable":false,"input":"转折"},"tag-232765":{"x":4010,"y":4199,"editable":false,"input":"看首句"},"tag-254375":{"x":4199,"y":4203,"editable":false,"input":"看尾句"},"tag-300172":{"x":4350,"y":4196,"editable":false,"input":"看答案，代入验证"},"tag-324368":{"x":4541,"y":4203,"editable":false,"input":"看转折之后"}},"LineGroup":{"line-72879":{"fromKey":"block-72108","toKey":"block-22683","from":{"x":420,"y":257,"width":100,"height":80,"top":257,"right":520,"bottom":337,"left":420},"to":{"x":855,"y":13,"width":100,"height":80,"top":13,"right":955,"bottom":93,"left":855}},"line-392146":{"fromKey":"block-391416","toKey":"block-72108","from":{"x":184,"y":431,"width":100,"height":80,"top":431,"right":284,"bottom":511,"left":184},"to":{"x":420,"y":257,"width":100,"height":80,"top":257,"right":520,"bottom":337,"left":420}},"line-406385":{"fromKey":"block-404318","toKey":"block-72108","from":{"x":626,"y":430,"width":100,"height":80,"top":430,"right":726,"bottom":510,"left":626},"to":{"x":420,"y":257,"width":100,"height":80,"top":257,"right":520,"bottom":337,"left":420}},"line-621502":{"fromKey":"block-620761","toKey":"block-404318","from":{"x":453,"y":610,"width":100,"height":80,"top":610,"right":553,"bottom":690,"left":453},"to":{"x":626,"y":430,"width":100,"height":80,"top":430,"right":726,"bottom":510,"left":626}},"line-626782":{"fromKey":"block-404318","toKey":"block-624523","from":{"x":626,"y":430,"width":100,"height":80,"top":430,"right":726,"bottom":510,"left":626},"to":{"x":606,"y":610,"width":100,"height":80,"top":610,"right":706,"bottom":690,"left":606}},"line-678046":{"fromKey":"block-677164","toKey":"block-404318","from":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756},"to":{"x":626,"y":430,"width":100,"height":80,"top":430,"right":726,"bottom":510,"left":626}},"line-757256":{"fromKey":"block-756260","toKey":"block-677164","from":{"x":439,"y":798,"width":100,"height":80,"top":798,"right":539,"bottom":878,"left":439},"to":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756}},"line-760372":{"fromKey":"block-759655","toKey":"block-677164","from":{"x":556,"y":799,"width":100,"height":80,"top":799,"right":656,"bottom":879,"left":556},"to":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756}},"line-770405":{"fromKey":"block-677164","toKey":"block-768369","from":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756},"to":{"x":675,"y":798,"width":100,"height":80,"top":798,"right":775,"bottom":878,"left":675}},"line-780794":{"fromKey":"block-779054","toKey":"block-677164","from":{"x":313,"y":799,"width":100,"height":80,"top":799,"right":413,"bottom":879,"left":313},"to":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756}},"line-787301":{"fromKey":"block-785673","toKey":"block-677164","from":{"x":809,"y":798,"width":100,"height":80,"top":798,"right":909,"bottom":878,"left":809},"to":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756}},"line-832364":{"fromKey":"block-830615","toKey":"block-677164","from":{"x":186,"y":798,"width":100,"height":80,"top":798,"right":286,"bottom":878,"left":186},"to":{"x":756,"y":612,"width":100,"height":80,"top":612,"right":856,"bottom":692,"left":756}},"line-907903":{"fromKey":"block-906652","toKey":"block-22683","from":{"x":1280,"y":256,"width":100,"height":80,"top":256,"right":1380,"bottom":336,"left":1280},"to":{"x":855,"y":13,"width":100,"height":80,"top":13,"right":955,"bottom":93,"left":855}},"line-5671":{"fromKey":"block-4900","toKey":"block-906652","from":{"x":996,"y":426,"width":100,"height":80,"top":426,"right":1096,"bottom":506,"left":996},"to":{"x":1280,"y":256,"width":100,"height":80,"top":256,"right":1380,"bottom":336,"left":1280}},"line-152343":{"fromKey":"block-151421","toKey":"block-906652","from":{"x":1181,"y":427,"width":100,"height":80,"top":427,"right":1281,"bottom":507,"left":1181},"to":{"x":1280,"y":256,"width":100,"height":80,"top":256,"right":1380,"bottom":336,"left":1280}},"line-170112":{"fromKey":"block-169247","toKey":"block-906652","from":{"x":1362,"y":427,"width":100,"height":80,"top":427,"right":1462,"bottom":507,"left":1362},"to":{"x":1280,"y":256,"width":100,"height":80,"top":256,"right":1380,"bottom":336,"left":1280}},"line-199478":{"fromKey":"block-198629","toKey":"block-906652","from":{"x":1540,"y":427,"width":100,"height":80,"top":427,"right":1640,"bottom":507,"left":1540},"to":{"x":1280,"y":256,"width":100,"height":80,"top":256,"right":1380,"bottom":336,"left":1280}},"line-218303":{"fromKey":"block-212716","toKey":"block-4900","from":{"x":994,"y":610,"width":100,"height":80,"top":610,"right":1094,"bottom":690,"left":994},"to":{"x":996,"y":426,"width":100,"height":80,"top":426,"right":1096,"bottom":506,"left":996}},"line-244819":{"fromKey":"block-151421","toKey":"block-243049","from":{"x":1181,"y":427,"width":100,"height":80,"top":427,"right":1281,"bottom":507,"left":1181},"to":{"x":1179,"y":613,"width":100,"height":80,"top":613,"right":1279,"bottom":693,"left":1179}},"line-295406":{"fromKey":"block-294569","toKey":"block-169247","from":{"x":1360,"y":617,"width":100,"height":80,"top":617,"right":1460,"bottom":697,"left":1360},"to":{"x":1362,"y":427,"width":100,"height":80,"top":427,"right":1462,"bottom":507,"left":1362}},"line-315077":{"fromKey":"block-314275","toKey":"block-198629","from":{"x":1538,"y":616,"width":100,"height":80,"top":616,"right":1638,"bottom":696,"left":1538},"to":{"x":1540,"y":427,"width":100,"height":80,"top":427,"right":1640,"bottom":507,"left":1540}}}}
