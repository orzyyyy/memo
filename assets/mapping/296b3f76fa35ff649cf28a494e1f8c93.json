{"CanvasPosition":{"x":-1262,"y":-1005,"z":0,"gap":1},"BlockGroup":{"block-357197":{"x":1944,"y":1138},"block-775102":{"x":1942,"y":1262},"block-490345":{"x":1735,"y":1443},"block-585848":{"x":2140,"y":1444},"block-133601":{"x":1946,"y":1009},"block-210595":{"x":1932,"y":1642},"block-718490":{"x":1761,"y":1830},"block-720034":{"x":2104,"y":1829}},"TagGroup":{"tag-660600":{"x":1965,"y":1036,"editable":false,"input":"制衡最大化"},"tag-789245":{"x":1979,"y":1291,"editable":false,"input":"平等"},"tag-498003":{"x":1736,"y":1468,"editable":false,"input":"充分制衡的权利下渗"},"tag-528747":{"x":1819,"y":1366,"editable":false,"input":"设置权利底限"},"tag-594707":{"x":2144,"y":1461,"editable":false,"input":"社会经济地位平等"},"tag-622842":{"x":2102,"y":1364,"editable":false,"input":"精英治国"},"tag-141574":{"x":1979,"y":1167,"editable":false,"input":"自由"},"tag-218588":{"x":1969,"y":1672,"editable":false,"input":"民主"},"tag-739078":{"x":1842,"y":1744,"editable":false,"input":"向下的自由"},"tag-747713":{"x":1763,"y":1854,"editable":false,"input":"民粹面目的极权主义"},"tag-788800":{"x":2034,"y":1744,"editable":false,"input":"向上的自由"},"tag-798930":{"x":2106,"y":1849,"editable":false,"input":"能提升多数公民下限的手段"}},"LineGroup":{"line-777472":{"fromKey":"block-775102","toKey":"block-357197","from":{"x":681,"y":258,"width":100,"height":80,"top":258,"right":781,"bottom":338,"left":681},"to":{"x":683,"y":134,"width":100,"height":80,"top":134,"right":783,"bottom":214,"left":683}},"line-491333":{"fromKey":"block-490345","toKey":"block-775102","from":{"x":474,"y":439,"width":100,"height":80,"top":439,"right":574,"bottom":519,"left":474},"to":{"x":681,"y":258,"width":100,"height":80,"top":258,"right":781,"bottom":338,"left":681}},"line-587256":{"fromKey":"block-585848","toKey":"block-775102","from":{"x":879,"y":440,"width":100,"height":80,"top":440,"right":979,"bottom":520,"left":879},"to":{"x":681,"y":258,"width":100,"height":80,"top":258,"right":781,"bottom":338,"left":681}},"line-134605":{"fromKey":"block-133601","toKey":"block-357197","from":{"x":685,"y":5,"width":100,"height":80,"top":5,"right":785,"bottom":85,"left":685},"to":{"x":683,"y":134,"width":100,"height":80,"top":134,"right":783,"bottom":214,"left":683}},"line-211538":{"fromKey":"block-210595","toKey":"block-585848","from":{"x":671,"y":638,"width":100,"height":80,"top":638,"right":771,"bottom":718,"left":671},"to":{"x":879,"y":440,"width":100,"height":80,"top":440,"right":979,"bottom":520,"left":879}},"line-213093":{"fromKey":"block-490345","toKey":"block-210595","from":{"x":474,"y":439,"width":100,"height":80,"top":439,"right":574,"bottom":519,"left":474},"to":{"x":671,"y":638,"width":100,"height":80,"top":638,"right":771,"bottom":718,"left":671}},"line-720845":{"fromKey":"block-720034","toKey":"block-210595","from":{"x":843,"y":825,"width":100,"height":80,"top":825,"right":943,"bottom":905,"left":843},"to":{"x":671,"y":638,"width":100,"height":80,"top":638,"right":771,"bottom":718,"left":671}},"line-724696":{"fromKey":"block-718490","toKey":"block-210595","from":{"x":500,"y":826,"width":100,"height":80,"top":826,"right":600,"bottom":906,"left":500},"to":{"x":671,"y":638,"width":100,"height":80,"top":638,"right":771,"bottom":718,"left":671}}}}
