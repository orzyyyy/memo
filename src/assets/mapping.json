[
  {
    "id": "b07b876e6050e333988404e268921906",
    "title": "js 基本类型",
    "url": "./assets/mapping/b07b876e6050e333988404e268921906.json",
    "createTime": 1558690403907,
    "modifyTime": 1558690403907,
    "type": "编程",
    "subType": "JavaScript",
    "category": "mapping"
  },
  {
    "id": "c9abecfb05549080298f37daea8de46f",
    "title": "页面加载优化",
    "url": "./assets/mapping/c9abecfb05549080298f37daea8de46f.json",
    "createTime": 1558690403907,
    "modifyTime": 1558690403907,
    "type": "编程",
    "subType": "JavaScript",
    "category": "mapping"
  },
  {
    "id": "e6342ea80e1b9da29570d49af1daaaeb",
    "title": "react 渲染过程",
    "url": "./assets/mapping/e6342ea80e1b9da29570d49af1daaaeb.json",
    "createTime": 1558690403907,
    "modifyTime": 1558690403907,
    "type": "编程",
    "subType": "react",
    "category": "mapping"
  },
  {
    "id": "f3a00b390b07d089f0ee965ecfa1354c",
    "title": "《民主的细节》摘抄",
    "url": "./assets/mapping/f3a00b390b07d089f0ee965ecfa1354c.json",
    "createTime": 1559178666618,
    "modifyTime": 1559779697256,
    "type": "读书",
    "subType": "摘抄",
    "category": "markdown"
  },
  {
    "id": "296b3f76fa35ff649cf28a494e1f8c93",
    "title": "《民主的细节》阅读思路整理",
    "url": "./assets/mapping/296b3f76fa35ff649cf28a494e1f8c93.json",
    "createTime": 1559351098071,
    "modifyTime": 1559372939631,
    "type": "读书",
    "subType": "思路",
    "category": "mapping"
  },
  {
    "id": "32e89f38116080d6b7608cf2137f3c96",
    "title": "音程表",
    "url": "./assets/mapping/32e89f38116080d6b7608cf2137f3c96.json",
    "createTime": 1559522075108,
    "modifyTime": 1559523700429,
    "type": "音乐",
    "subType": "乐理",
    "category": "markdown"
  },
  {
    "id": "7a73d21c13f1eb55053d4af66e0d057b",
    "title": "windows 下安装 rust",
    "url": "./assets/mapping/7a73d21c13f1eb55053d4af66e0d057b.json",
    "createTime": 1559617097664,
    "modifyTime": 1559633639037,
    "type": "编程",
    "subType": "Rust",
    "category": "markdown"
  },
  {
    "id": "57ca721de1c14fb189d8bc5f6e14448c",
    "title": "改 cookie 步骤",
    "url": "./assets/mapping/57ca721de1c14fb189d8bc5f6e14448c.json",
    "createTime": 1560220086082,
    "modifyTime": 1560220318066,
    "type": "编程",
    "subType": "ExHentai",
    "category": "markdown"
  },
  {
    "id": "56f14e6c24b6ece2e63bdf56accdef16",
    "title": "爬虫思路",
    "url": "./assets/mapping/56f14e6c24b6ece2e63bdf56accdef16.json",
    "createTime": 1560392150068,
    "modifyTime": 1560392654377,
    "type": "编程",
    "subType": "ExHentai",
    "category": "markdown"
  },
  {
    "id": "2b91499cc79105eef0a71f5a3c7df3b7",
    "title": "单词积累",
    "url": "./assets/mapping/2b91499cc79105eef0a71f5a3c7df3b7.json",
    "createTime": 1560473744117,
    "modifyTime": 1561955003623,
    "type": "公考",
    "subType": "言语理解与表达",
    "category": "markdown"
  },
  {
    "id": "ac1b811c06101cf7cd870495cb6d6662",
    "title": "词语填空解题思路",
    "url": "./assets/mapping/ac1b811c06101cf7cd870495cb6d6662.json",
    "createTime": 1560493141920,
    "modifyTime": 1561108324553,
    "type": "公考",
    "subType": "言语理解与表达",
    "category": "mapping"
  },
  {
    "id": "d84c686eca77b9db11063f995d4f4694",
    "title": "deploy with gh-pages and circleci",
    "url": "./assets/mapping/d84c686eca77b9db11063f995d4f4694.json",
    "createTime": 1560559934239,
    "modifyTime": 1563003475889,
    "type": "编程",
    "subType": "其他",
    "category": "markdown"
  },
  {
    "id": "edabfb3a75ee39f82ad29e384d04dc0b",
    "title": "词语填空错题集",
    "url": "./assets/mapping/edabfb3a75ee39f82ad29e384d04dc0b.json",
    "createTime": 1560583719588,
    "modifyTime": 1560931062586,
    "type": "公考",
    "subType": "言语理解与表达",
    "category": "markdown"
  },
  {
    "id": "9c44d7d3145e382d441fe81ef477e4c6",
    "title": "片段阅读错题集",
    "url": "./assets/mapping/9c44d7d3145e382d441fe81ef477e4c6.json",
    "createTime": 1561015817211,
    "modifyTime": 1561277782899,
    "type": "公考",
    "subType": "言语理解与表达",
    "category": "markdown"
  },
  {
    "id": "2250bfcdc903daabb68a9925e8701383",
    "title": "片段阅读考点",
    "url": "./assets/mapping/2250bfcdc903daabb68a9925e8701383.json",
    "createTime": 1561249847278,
    "modifyTime": 1561280230101,
    "type": "公考",
    "subType": "言语理解与表达",
    "category": "mapping"
  },
  {
    "id": "5652a52ab188e25bddf687b32d4ce326",
    "title": "linux after install",
    "url": "./assets/mapping/5652a52ab188e25bddf687b32d4ce326.json",
    "createTime": 1561623131393,
    "modifyTime": 1563003318050,
    "type": "编程",
    "subType": "linux",
    "category": "markdown"
  },
  {
    "id": "b921eb06421a6a25e75c8009721d7324",
    "title": "github ssh proxy",
    "url": "./assets/mapping/b921eb06421a6a25e75c8009721d7324.json",
    "createTime": 1561705415318,
    "modifyTime": 1561705415318,
    "type": "编程",
    "subType": "linux",
    "category": "markdown"
  },
  {
    "createTime": 1562317110304,
    "modifyTime": 1562317700285,
    "url": "./assets/mapping/91ab070449395f41409a2dd206c25c6b.json",
    "id": "91ab070449395f41409a2dd206c25c6b",
    "title": "settings sync 同步",
    "type": "编程",
    "subType": "其他",
    "category": "markdown"
  },
  {
    "createTime": 1562373606865,
    "modifyTime": 1562374384736,
    "url": "./assets/mapping/75bb0cdbfd5e9961635470f42b66c92d.json",
    "id": "75bb0cdbfd5e9961635470f42b66c92d",
    "title": "到底怎样才不算是搬砖、API Caller、Crud Boy？",
    "type": "公考",
    "subType": "申论练习",
    "category": "markdown"
  },
  {
    "createTime": 1563676191249,
    "modifyTime": 1563847993715,
    "url": "./assets/markdown/e80f3dfbe1ec6675f0bd453d4beb081a.json",
    "id": "e80f3dfbe1ec6675f0bd453d4beb081a",
    "title": "常用解题算法",
    "type": "公考",
    "subType": "资料分析",
    "category": "markdown"
  }
]
