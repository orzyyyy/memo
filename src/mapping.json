[{"id":"1a79a4d60de6718e8e5b326e338ae533","title":"test1","thumbnailUrl":"dist/lib/main/assets/5795774_0.jpg","hoverText":"测试 layout-1a79a4d60de6718e8e5b326e338ae533.json","createTime":1548562687561,"modifyTime":1548562687561},{"id":"1a79a4d60de6718e8e5b326e338ae534","title":"test1","thumbnailUrl":"dist/lib/main/assets/5795774_0.jpg","hoverText":"测试 layout-1a79a4d60de6718e8e5b326e338ae534.json","createTime":1548562687561,"modifyTime":1548562687561}]