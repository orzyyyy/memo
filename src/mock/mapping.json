{
  "BlockGroup": {
    "block-623187": {
      "x": 158,
      "y": 256,
      "style": {
        "width": 100,
        "height": 80,
        "background": "#F96",
        "borderRadius": "10px",
        "border": "1px solid #aaa"
      }
    },
    "block-624018": {
      "x": 367,
      "y": 368,
      "style": {
        "width": 100,
        "height": 80,
        "background": "#F96",
        "borderRadius": "10px",
        "border": "1px solid #aaa"
      }
    },
    "block-73377": {
      "x": 253,
      "y": 525,
      "style": {
        "width": 100,
        "height": 80,
        "background": "#F96",
        "borderRadius": "10px",
        "border": "1px solid #aaa"
      }
    }
  },
  "TagGroup": {
    "tag-626505": {
      "x": 167,
      "y": 284,
      "style": { "width": 100, "height": 32 },
      "editable": false,
      "input": "test"
    },
    "tag-629962": {
      "x": 405,
      "y": 398,
      "style": { "width": 100, "height": 32 },
      "editable": false,
      "input": "test2"
    },
    "tag-80986": {
      "x": 286,
      "y": 555,
      "style": { "width": 100, "height": 32 },
      "editable": false,
      "input": "test3"
    }
  },
  "LineGroup": {
    "line-77619": { "fromKey": "block-73377", "toKey": "block-623187" },
    "line-592694": { "fromKey": "block-623187", "toKey": "block-624018" }
  }
}
