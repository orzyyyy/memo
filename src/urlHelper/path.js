/*eslint-disable */
const proxy = null;
const isDev = process.env.NODE_ENV === 'development';

const prod = {
  propertyDatas: '../../mock/propertyDatas.json',
  s_slmh_menu_data:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=s_slmh_menu_data',
  s_slmh_meal_layout_data:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=s_slmh_meal_layout_data',
  s_slmh_meal_switch:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=s_slmh_meal_switch',
  d_slmh_meal:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=d_slmh_meal',
  add_meal: 'http://47.95.1.229:9007/webapi1/api/basic/meal',
  style_menu_data: '../../mock/menuDatas.json',
  'login-ticket': 'http://47.95.1.229:9007/webapi/api/v1/zjzhsl/login-ticket',
};

const dev = {
  propertyDatas: '../../mock/propertyDatas.json',
  s_slmh_menu_data:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=s_slmh_menu_data',
  s_slmh_meal_layout_data:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=s_slmh_meal_layout_data',
  s_slmh_meal_switch:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=s_slmh_meal_switch',
  d_slmh_meal:
    'http://47.95.1.229:9003/webapi/api/v1.1/basic/data?key=d_slmh_meal',
  add_meal: 'http://47.95.1.229:9007/webapi1/api/basic/meal',
  style_menu_data: '../../mock/menuDatas.json',
  'login-ticket': 'http://47.95.1.229:9007/webapi/api/v1/zjzhsl/login-ticket',
};

const path = isDev ? dev : prod;

export { path, proxy, isDev };
