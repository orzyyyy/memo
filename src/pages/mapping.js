import React, { Component } from 'react';

import '../assets/mapping.css';

export default class mapping extends Component {
  static propTypes = {};

  static defaultProps = {};

  constructor(props) {
    super(props);

    this.state = {};
  }

  componentDidMount = () => {
    // eslint-disable-next-line
    console.log(location);
  };

  render = () => {
    return <div className="mapping">mapping</div>;
  };
}
