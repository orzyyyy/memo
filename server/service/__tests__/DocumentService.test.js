import Service from '../DocumentService';
import path from 'path';
import mapping from '../../../src/assets/mapping.json';

describe('DocumentService', () => {
  let service;

  beforeEach(() => {
    service = new Service();
  });

  afterEach(() => {
    service = null;
  });

  it('updateMapping', () => {
    let result = service.updateMapping({
      id: 'b07b876e6050e333988404e268921906',
      test: undefined,
      category: 'mapping',
    });
    expect(result).toEqual(mapping);
    result = service.updateMapping(
      {
        id: 'b07b876e6050e333988404e268921906',
        test: undefined,
        category: 'mapping',
      },
      true,
    );
    expect(
      result.filter(item => item.id === 'b07b876e6050e333988404e268921906'),
    ).toEqual([]);
  });

  it('getOriginContent', () => {
    let result = service.getOriginContent(
      path.join(__dirname, '../../../src/assets/mapping.json'),
      null,
      true,
    );
    expect(result).toEqual(mapping);
    result = service.getOriginContent(
      path.join(__dirname, '../../../src/assets/mapping.json'),
      'layout',
      true,
    );
    expect(result).toBe('layout');
  });

  it('initHtmlTemplate', () => {
    const result = service.initHtmlTemplate(
      'markdown',
      '7a9a2d738fa682b6c2b1abc0471616b2',
    );
    expect(result).toBe(
      path.join(
        __dirname,
        '../../../',
        'dist/markdown/84a65e108e3721e07261e6b3c4336ff1/index.html', // id comes from ../__mocks__/fs-extra.js
      ),
    );
  });

  it('updateContent', () => {
    expect(
      service.updateContent('markdown', ['7a9a2d738fa682b6c2b1abc0471616b2']),
    ).toMatchSnapshot();
    expect(
      service.updateContent('mapping', ['7a9a2d738fa682b6c2b1abc0471616b2']),
    ).toMatchSnapshot();
  });

  it('deleteTargetDocument', () => {
    try {
      service.deleteTargetDocument(['7a9a2d738fa682b6c2b1abc0471616b2']);
    } catch (error) {
      expect(error.message).toBe(
        "7a9a2d738fa682b6c2b1abc0471616b2 doesn't exist.",
      );
    }
    try {
      service.deleteTargetDocument([
        path.join(
          process.cwd(),
          'src/assets/markdown/7a9a2d738fa682b6c2b1abc0471616b2.md',
        ),
      ]);
    } catch (error) {
      expect(error).toBe();
    }
  });
});
