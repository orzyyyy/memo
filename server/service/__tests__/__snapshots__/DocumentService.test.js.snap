// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentService initHtmlTemplate 1`] = `"/home/orzorzorzorz/github/memo/dist/markdown/84a65e108e3721e07261e6b3c4336ff1/index.html"`;

exports[`DocumentService updateContent 1`] = `undefined`;

exports[`DocumentService updateContent 2`] = `undefined`;
